spec/helpers/comments_helper_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper'
+
+describe CommentsHelper do
+  describe "#comment_path" do
+    let(:comment) { VCR.use_cassette('planningalerts') { create(:confirmed_comment) } }
+
+    it "returns the path for the application with an anchor for the comment" do
+      expect(helper.comment_path(comment)).to eq application_path(comment.application, anchor: "comment#{comment.id}")
```

Comment:
This test is generating both the test and the expectation. The expectation shouldn't be generated, instead it should just be a string, so that you're actually testing the generation properly.